field.py

```diff
@@ -64,11 +64,11 @@ def SetFieldRadioButtons(self):
         
         self.NewField(self.Vfield, self.Vfield.value)  # setChecked the current field
```

Comment:
L65 - Should this read ```python self.NewField(self.Vfield, self.Vfield.value, False)``` ?
This seems like a case where a weak change would be applicable?